debian/rules

```diff
@@ -6,6 +6,8 @@ include /usr/share/rustc/architecture.mk
 # The package name for authd, used only locally to avoid repetitions
 AUTHD_GO_PACKAGE := $(shell grep-dctrl -s XS-Go-Import-Path -n - ./debian/control)
 
+AUTHD_GO_VERSION := $(shell grep-dctrl -s Build-Depends -n - ./debian/control | sed -n "s,.*\bgolang-\([0-9.]\+\)\b.*,\1,p")
```

Comment:
As we don’t reuse that variable anywhere mind just putting it above PATH, without line break? That we, we keep that scoped. If we go with the alternative. However, we need to ideally only expose PATH if the go version in PATH doesn’t match the requirement.

Reply:
Ok done.